### ISSUE TITLE
A few of these sections need to be updated.

### ISSUE THREAD

**Issue by Sdcrouse** [learn-co-curriculum/sinatra-mechanics-of-sessions-readme#16]
Hi. I looked over the "Viewing in Chrome Settings" and "Clearing Sessions" sections, and I think both of them need to be updated.

My guess is that Chrome has been updated quite a bit since the last time the README was updated, so the interface is different that what was shown in the "Viewing in Chrome Settings" section. Here's what you have to do now after going to the `Settings`: instead of clicking on the `Show advanced settings` link, click on the `Advanced` drop-down tab (still at the bottom). Then, in the `Privacy and Security` section, click on `Site Settings`. Then, in the `Permissions` section, click on `Cookies`. Finally, click on `See all cookies and site data`, and you're good to go!

As for the "Clearing Sessions" section, I tried right-clicking on a session in the Developer Tools as the README said. However, the only option that pops up is `Refresh`. I don't know if there's a way to `Clear All from "learn.co"` anymore.

Thanks (as always) for looking into this!

---Sdcrouse

**Comment by alouiscious** [comment]
The current settings menus in chrome have change and make this readme hard to follow.
please update this to make it easier to follow the directions; especially those that reference the process to locate the cookie settings in the advanced setting menus in chrome.
